modules/Project/gantt.php

```diff
@@ -224,10 +249,10 @@ public function year_month($start_date, $end_date, $diff_interval=1)
 		$interval = new DateInterval('P1D'); // 1 month interval
         $period = new DatePeriod($begin, $interval, $end);
         $aResult = array();
-
+        
         foreach ( $period as $dt )
         {
-            $aResult[$dt->format('Y')][strftime("%B", $dt->getTimestamp())][$dt->format('j')] = strftime("%a", $dt->getTimestamp());
+        	$aResult[$dt->format('Y')][$GLOBALS['app_list_strings']['dom_cal_month_short'][$dt->format('n')]][$dt->format('j')] = $GLOBALS['app_list_strings']['dom_cal_day_short'][$dt->format('w')+1];
```

Comment:
Line is longer than 120 characters and it is also difficult to read.Consider assigning the $GLOBALS and $dt->format methods into variables and then use the variables instead.

```
public function <name>
{
    global $app_list_strings;
    // ... 
    foreach ($period as $dt) {
        // Perhaps give better names here
        $y = $dt->format('Y');
        $n = $app_list_strings[$dt->format('n')];
        $w = $dt->format('w') + 1;
        $j = $dt->format('j');
            
        $aResult[$y][$n][$j] = $app_list_strings['dom_cal_day_short'][$w];
    }
    // ...
}

```